metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -133,6 +133,9 @@ def step_init(obj, run_id, step_name, passed_in_split_indexes, task_id):
 @click.option(
     "--workflow-timeout", default=None, type=int, help="Workflow timeout in seconds."
 )
+@click.option(
+    "--workflow-ttl", default=None, type=int, help="Workflow time to live in seconds."
```

Comment:
Can you please add here in the help what the difference between timeout and ttl is?
I don't see the difference, unless I dive into the code.  This is TTL after finished, what does that mean, and how does it affect the Applied Scientist?

Also, do you want None to be the default, or do you want the default to be some timespan? Possibly pick this up from metaflow config?

Reply:
Good idea, I'll add some comments on what TTL is. I've specified the TTL elsewhere to 1 week, but now that you mention it, it would be easier to specify here.